README.md

```diff
@@ -18,10 +18,10 @@ let name = 'foo bar'
 await $`mkdir /tmp/${name}`
 ```
 
-Bash is great, but when it comes to writing scripts, 
+Bash is great, but when it comes to writing scripts,
 people usually choose a more convenient programming language.
-JavaScript is a perfect choice, but standard Node.js library 
-requires additional hassle before using. The `zx` package provides
+JavaScript is a perfect choice, but standard Node.js libraries
```

Comment:
It's one library which contains lots of packages.

Reply:
Got you, changed to "the Node.js standard library requires additional hassle..."